src/patternfly/components/Table/table-grid.scss

```diff
@@ -37,16 +37,16 @@
   // this is purposeful and necessary to avoid adding selectors to each td/th
   // ============================================================ //
 
-  // Responsive vars
-  // ==================================================================
-
   // Table
   --pf-c-table--responsive--BorderColor: var(--pf-global--BorderColor--300);
 
   // Body
   --pf-c-table-tbody--responsive--MarginTop: var(--pf-global--spacer--xs);
   --pf-c-table-tbody--m-expanded--before--Top: var(--pf-global--spacer--sm);
   --pf-c-table-tbody--responsive--BorderWidth: var(--pf-global--spacer--sm);
+  --pf-c-table--tbody--after--BorderWidth: var(--pf-global--BorderWidth--lg);
```

Comment:
should this be `pf-c-table--tbody` or `pf-c-table-tbody`, if we're going with `pf-c-table--tbody` looks like we should update it everywhere

Reply:
Great point! To limit the amount of potential breaks, I'll open another issue to update all variables to our current spec! https://github.com/patternfly/patternfly/issues/2983